snapfaas/bins/gateway/main.rs

```diff
@@ -0,0 +1,77 @@
+use clap::{App, Arg, crate_authors, crate_version};
```

Comment:
Sorry for being very detailed but can we rename this binary to scheduler?

Reply:
Fixed